packages/nuget/Sentry.AspNet/4.2.1.json

```diff
@@ -4,5 +4,30 @@
   "version": "4.2.1",
   "package_url": "https://www.nuget.org/packages/Sentry.AspNet",
   "repo_url": "https://github.com/getsentry/sentry-dotnet",
-  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnet"
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnet",
+  "categories": [
+    "server"
+  ],
+  "features": {
+    "crons": {
+      "availability": "GA",
+      "available-with-version": "4.2.0"
+    },
+    "errors": {
+      "availability": "GA",
+      "available-with-version": "1.0.0"
+    },
+    "performance": {
+      "availability": "GA",
+      "available-with-version": "3.0.0"
+    },
+    "profiling": {
+      "availability": "GA",
+      "available-with-version": "3.31.0"
+    },
+    "session-replay": {
```

Comment:
here we mark as "yes" with the loader. since it's a web backend project that can render web pages.